app/src/main/java/com/ds/avare/storage/Preferences.java

```diff
@@ -349,6 +349,10 @@ public float getAudibleTrafficAlertsMaxFrequency() {
         return Float.parseFloat(mPref.getString(mContext.getString(R.string.AudibleTrafficAlertsMaxFrequency), "15.0"));
     }
 
+    public float getAudibleTrafficAlertsMinSpeed() {
+        return Float.parseFloat(mPref.getString(mContext.getString(R.string.AudibleTrafficAlertsMinimumSpeed), "0.0"));
```

Comment:
Will this require a try catch, or is this by design always a number?

Reply:
Committed